lib/vsc/filesystem/quota/tools.py

```diff
@@ -234,104 +224,89 @@ def process_user_quota(storage, gpfs, storage_name, filesystem, quota_map, user_
     return exceeding_users
 
 
-def get_mmrepquota_maps(quota_map, storage, filesystem, filesets,
-                        replication_factor=1):
+def get_quota_maps(storage, storage_name):
     """Obtain the quota information.
 
-    This function uses vsc.filesystem.gpfs.GpfsOperations to obtain
+    This function uses the storage backend operator to obtain
     quota information for all filesystems known to the storage.
 
     The returned dictionaries contain all information on a per user
     and per fileset basis for the given filesystem. Users with multiple
     quota settings across different filesets are processed correctly.
 
     Returns { "USR": user dictionary, "FILESET": fileset dictionary}.
-
-    @type replication_factor: int, describing the number of copies the FS holds for each file
-    @type metadata_replication_factor: int, describing the number of copies the FS metadata holds for each file
     """
     user_map = {}
     fs_map = {}
 
     timestamp = int(time.time())
 
-    logging.info("ordering USR quota for storage %s", storage)
-    # Iterate over a list of named tuples -- GpfsQuota
-    for (user, gpfs_quota) in quota_map['USR'].items():
-        user_quota = user_map.get(user, QuotaUser(storage, filesystem, user))
+    filesystem = storage[storage_name].filesystem
+
+    fs_backend = load_storage_operator(storage[storage_name])
+    quotas = fs_backend.list_quota(devices=filesystem)
+    quota_map = quotas[filesystem]
+
+    logging.info("ordering USR quota for storage %s", storage_name)
```

Comment:
`USR` -> some enum.value

Reply:
fixed in https://github.com/hpcugent/vsc-filesystems-quota/pull/19/commits/03e2f3573385020373e31c9fe977bac475c6d667